sdk/cmake/gcc.cmake

```diff
@@ -102,6 +102,7 @@ add_compile_options(-Wno-char-subscripts -Wno-multichar -Wno-unused-value)
 add_compile_options(-Wno-unused-const-variable)
 add_compile_options(-Wno-unused-local-typedefs)
 add_compile_options(-Wno-deprecated)
+add_compile_options(-Wno-unused-result) # FIXME To be removed when CORE-17637 is resolved
```

Comment:
What about allowing this warning instead of disabling it?

Reply:
I enable it for rtl & ntoskrnl. There 350 more occurrences to fix, I'd propose a per subdirectory approach. I already started with base/applications